cypress/support/commands.ts

```diff
@@ -69,3 +90,14 @@ Cypress.Commands.add("checkColorContrast", () => {
     cy.injectAxe();
     cy.checkA11y(undefined, { runOnly: { type: "tag", values: ["wcag2aa"] } }, terminalLog);
```

Comment:
How come we're not using `cy.checkAccessibility`

Reply:
It's used in accessibility.cy.ts